tck/src/main/java/org/eclipse/microprofile/lra/tck/TckParticipantTests.java

```diff
@@ -142,6 +142,9 @@ public void testNonJaxRsCompletionStageVoid() throws InterruptedException {
         URI lraId = URI
                 .create(checkStatusReadAndCloseResponse(Response.Status.INTERNAL_SERVER_ERROR, response, resourcePath));
 
+        // lraTestService.waitForCallbacks(lraId);
+        lraTestService.waitForRecovery(lraId);
```

Comment:
Seems like forgotten testing? Decide if it should be waitForCallbacks (I think it should) or waitForRecovery.

Reply:
Sure, let me change it to waitForCallbacks()